web/modules/custom/analysis/analysis.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Analysis
+type: module
+core_version_requirement: ^9 || ^10
+description: 'Try-out to validate the dependencies are include or not.'
+package: Custom
+dependencies:
```

Comment:
Is this dependency required?

Reply:
I was exploring the dependency since the ticket included understanding the dependency in the info.yml file.